src/views/shared/private-route/private-route.view.tsx

```diff
@@ -5,11 +5,13 @@ import { useProvidersContext } from "src/contexts/providers.context";
 import routes from "src/routes";
 
 const PrivateRoute: FC<PropsWithChildren> = ({ children }) => {
-  const { connectedProvider: provider } = useProvidersContext();
-  const { pathname } = useLocation();
+  const { connectedProvider } = useProvidersContext();
+  const { pathname, search } = useLocation();
```

Comment:
I think that now the `search` doesn't make sense as we are not using query params right?